src/main/java/org/gridsuite/securityanalysis/server/SecurityAnalysisController.java

```diff
@@ -118,4 +118,21 @@ private static SecurityAnalysisParameters getNonNullParameters(SecurityAnalysisP
         Mono<Void> result = service.deleteResults();
         return ResponseEntity.ok().body(result);
     }
+
+    @GetMapping(value = "/status/{resultUuid}", produces = APPLICATION_JSON_VALUE)
```

Comment:
To be consistent with the other /results/{resultUuid}/status ?

Reply:
Corrected